test/nexus-integration.py

```diff
@@ -134,6 +135,63 @@ def test_search_nexus3_pyversion(self):
         )
         self.assertListEqual(versions, [pipio.Version('1.0.0'), pipio.Version('1.0.1rc1'), pipio.Version('1.0.1')])        
 
+class TestPut(unittest.TestCase):
+
+    def test_upload_package_for_pep440_compliant_version(self):
+        pkg_name = 'test_package2' # must differ from package names used in TestCheck
```

Comment:
One way to make it clear is to use 'check_step_test_package' for TestCheck and 'put_step_test_package' for this test.

Reply:
> One way to make it clear is to use 'check_step_test_package' for TestCheck and 'put_step_test_package' for this test.

I will replace `put_step_test_package`, it will be clearer at least in this test. Thanks
I can not actually change it in the TestCheck phase for now because of how convoluted this is with `make test` command (unnecessary and wrong). I actually tried a lot to refactor all of the tests, and I think I can bring a lot of improvements here.  
- don't build test packages with make.
- build test packages using tmpfiles (separation and it conflicting files from different tests in the same folder) and a single generalize package template to reduce fixture files duplication
- ensure the files are cleaned up after each tests 

But it would be outside of the scope of this PR.

Reply:
To address the comment, I have changed the package name to: `put_step_test_package`. As explained about this is not easily done in for TestCheck. It would require the large refactoring that should not be part of this PR.